domains/Users/Users/Observers/UsersObserver.php

```diff
@@ -19,8 +34,6 @@ class UsersObserver
     public function saving(Users $user) : void
```

Comment:
maybe we need to change this to `creating` since saving also  includes updating.